legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure

```diff
@@ -157,7 +157,7 @@ function meta::analytics::search::transformation::mapping::buildPropertyList(pro
 function meta::analytics::search::transformation::service::buildDocument(service: meta::legend::service::metamodel::Service[1]): meta::analytics::search::metamodel::service::RootDocument[1]
 {
   ^meta::analytics::search::metamodel::service::RootDocument(
-    id = $service.pattern,
+    id = $service->elementToPath(),
```

Comment:
This makes sense to be the ID, but should this really be extending from the packageable element root document and have a specific project for element path